bagua/service/autotune_service.py

```diff
@@ -241,12 +241,13 @@ def register_models():
                     10 * 1024 ** 5
                 )  # if you don't divide buckets, set big bucket as 10PB.
             session["bucket_size"] = self.sess_bucket_size
+
             with self.ask_hyperparameters_mutex:
                 self.param_group_info = param_group_info
                 default_hyperparameters = BaguaHyperparameter(
                     buckets=split_bucket_by_bucket_size(
                         tensor_list,
-                        self.default_bucket_size,
+                        session["bucket_size"],
```

Comment:
is this a bug?

Reply:
Yes, after the fix, the performance of decentralized is better.

Reply:
submit bugfix in separate PR and mark `fix:` in the title

such commits message will go to release notes

Reply:
OK